### PR TITLE
Filter logs

### PR DESCRIPTION
**Pull request by felisity09**
- Filter for time, keyword and level

### CODE REVIEW

File: server/controllers/filterLogController.js [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+
+const filterLogController = {};
+
+filterLogController.filter = async (req, res, next) => {
+  try {
+    const { level, startSearch, keyword } = req.body;
+
+    if (!level && !startSearch && !keyword) {
+      const dataError = {
+        log: 'filterLogController middleware: no target for filtering',
```
Comment (Agfeynman) [inline server/controllers/filterLogController.js:11]: This should be controlled on the FE and also where the filters are empy here we should return all logs by default.

File: server/controllers/filterLogController.js [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+        status: 406,
+        message: { err: 'An error occurred' },
+      };
+      return next(dataError);
+    }
+
+    res.locals.filteredLogs = data;
+    return next();
+  } catch (err) {
+    console.log(`error from filter log. Message: ${err}`);
```
Comment (Agfeynman) [inline server/controllers/filterLogController.js:48]: - We should use our custom logger
- That should be managed by our globar error handler

**Review by Agfeynman — APPROVED** [review]
Run the linter and push it again pls.

**Comment by felisity09** [comment]
All comments have been adjudicated. New refracted code is to wait till after MVP to merge into dev.

**Pull request closed without merging**